lib/oplog/tail.go

```diff
@@ -505,19 +543,14 @@ func (tailer *Tailer) parseRawOplogEntry(entry rawOplogEntry, txIdx *uint) []opl
 
 		out.Database, out.Collection = parseNamespace(out.Namespace)
 
+		var errID error
```

Comment:
Nit: What about calling this just `err`? `errID` sounds like it stores an error ID, rather than it being the error for the ID parsing part.

Reply:
Yeah, this is a pattern Yotam likes -- to be clear what each error variable is for, rather than just always `err`.  But I'm fine either way, and I agree this could be confusing.